CONTRIBUTING.md

```diff
@@ -1,10 +1,22 @@
 # Contributing
 
+## Introduction
+
+This book is a catalogue of Rust programming techniques, (anti-)patterns, and idioms and other explanations.
+It is a compilation of collective (sometimes implicit) knowledge or experience that was created with collaborative effort.
+
+These contained information are __no rules__, they should be understood as guidelines.
+We are collecting these information so people can form a balanced view on certain aspects.
```

Comment:
```suggestion
We are collecting Rust patterns in this book so people can learn Rust idioms and use them in their own code.
```

Reply:
I miss the balanced view. The point was more to give information to the people so they can be informed about how things could be done and in the end they can decide with more knowledge about how they use things in their code. Hence we're trying to often give advantages and disadvantages to patterns, idioms etc.

Do you have a recommendation how to keep the mindset of "informing for a balanced view" in that sentence but make it better? :-P

Reply:
```suggestion
We are collecting Rust patterns in this book so people can learn the tradeoffs between Rust idioms and use them properly in their own code.
```

Maybe something like this?
I'm fine with the wording being changed around a bit on this, I made the changes to add `Rust patterns` and `Rust idioms` being explicit here because it's more direct to the reader.